recipes/pydoe3/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pyDOE3" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|lower }}-{{ version }}.tar.gz
+  sha256: fc3210ea5a3c1059587e06f1b559bad664e2ff817f28a2bf794901f991ddd1d5
+
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - hatchling
```

Metadata has `hatch-vcs`. I don't know the first thing about hatch but maybe it is needed to add the version properly?

```suggestion
    - hatchling
    - hatch-vcs
```